lib/ProductOpener/Tags.pm

```diff
@@ -1302,6 +1302,36 @@ sub build_tags_taxonomy ($tagtype, $publish) {
 				}
 
 			}
+			elsif ($line =~ /^expected_nutriscore_grade:en:/) {
+				# the line should be the nutriscore grade: a, b, c, d or e
+				my $nutriscore_grade = $';    # everything after the matched string
+
+				if (not($nutriscore_grade =~ /^([a-e]){1}$/i)) {
+					my $msg
+						= "expected_nutriscore_grade:en: in "
+						. $tagtype
+						. " should be followed by a single letter between a and e. expected_nutriscore_grade:en: "
+						. $nutriscore_grade
+						. " is incorrect\n";
+
+					$errors .= "ERROR - " . $msg;
+				}
+			}
+			elsif ($line =~ /^expected_ingredients:en:/) {
+				# the line should contain a single ingredient
+				my $expected_ingredients = $';    # everything after the matched string
+
+				if ($expected_ingredients =~ /,/i) {
```

Comment:
@stephanegigandet, is there a function that could be used there to make sure that the ingredient provided is in the ingredients taxonomy?

Reply:
there's an exist_taxonomy_tag() function, but that would imply that the ingredients taxonomy would need to be loaded and built before the categories taxonomy.

Reply:
then, I will leave it as is for now